assembly/assembly-ide-war/src/main/resources/org/eclipse/che/ide/IDE.gwt.xml

```diff
@@ -93,4 +93,5 @@
     <!-- Specify the paths for translatable code                    -->
     <source path='client'/>
     <source path='shared'/>
+    <collapse-all-properties />
```

Comment:
right, I need to remove that line. I added it as part of the trick to reduce the build time during dev, but this should not have been included in this patch.